### ISSUE TITLE
The linked docs page is unusable

### ISSUE THREAD

**Issue by davidsusu** [rjsf-team/react-jsonschema-form#3113]
In the main README.md there is a link: "Explore the docs »". It links to https://react-jsonschema-form.readthedocs.io/en/stable/ which is currently not working. For each sub-page I get a 404 error. Most assets does not load, the design is strange.

Probably, it would be better to use the `docs` directory as a documentation.

**Comment by davidsusu** [comment]
Also, in `docs/index.md` there is a link to `usage/themes` ("Using Themes") which doesn't exists (missing the `.md` suffix).

**Comment by heath-freenome** [comment]
@davidsusu Yes, we are struggling to get the stable pages to work... for now, switch `stable` to `latest` and you should at least see the formatted information, which may be "ahead" of the last release.
